src/args.rs

```diff
@@ -33,11 +33,19 @@ pub struct Args {
     #[arg(short, long)]
     pub file: Option<String>,
 
+    /// The model to use. Defaults to `gpt-4`
+    #[arg(short = 'o', long)]
```

Comment:
🔧 Lets stick with `-m` for the short name here,  `-o` is commonly used by compilers to denote output and also clap defaults to the first letter of the long command for the short name.
```suggestion
    #[arg(short, long)]
```

Reply:
Some other options:

1. We drop -m for markdown altogether
2. We make model a capital M so it will end up being `cgip -M gtp-4-turbo`

Reply:
hmm it doesnt seem to like capital M ... I dont understand why though
```
error[E0277]: the trait bound `&str: IntoResettable<char>` is not satisfied
   --> src/args.rs:37:19
    |
37  |     #[arg(short = "M", long)]
    |           -----   ^^^ the trait `IntoResettable<char>` is not implemented for `&str`
    |           |
    |           required by a bound introduced by this call
```

Reply:
Went with option 2 here in latest PR. Think -M works